modules/storage-persistent-storage-pv.adoc

```diff
@@ -278,6 +280,7 @@ The following PV types support mount options:
 - iSCSI
 - Local volume
 - NFS
+- OpenStack Manila
```

Comment:
Does Manila support mount options?

Reply:
@bobfuru Did not find manila support mount options, need @Fedosin to double confirm.

Reply:
Removed manila from "support mount options" list. 
@Fedosin Correct that Manila does not support mount options?

Reply:
Per Mike: "users can't specify mount options for Manila"

Reply:
Yes, Manila doesn't support mount options